packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -45,6 +45,7 @@ export const comments: { readonly [key in TokenName]: string } = {
     modalBackdropColor: 'Color of background overlay behind modal dialog boxes',
     popupBorderColor: 'Border color for menus and dialog boxes',
     cardBorderColor: 'Border color for cards',
+    chipBackgroundColor: 'Background color for chips',
```

Comment:
Think I'd opt for `tagFillColor`

tag: openui working group resolved chip and pills as style variations of tags https://github.com/openui/open-ui/issues/266#issuecomment-1072843538

fill: seems more similar to a button type control and on buttons we refer to it as a fill